x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -214,11 +214,15 @@ export function ServiceMap({ serviceName }: ServiceMapProps) {
         width={width}
         style={cytoscapeDivStyle}
       >
-        <Controls />
-        {serviceName && renderedElements.current.length === 1 && (
-          <EmptyBanner />
-        )}
-        <Popover focusedServiceName={serviceName} />
+        {cy => {
```

Comment:
I think we should be using `const cy = useContext(CytoscapeContext)` rather than the function as children pattern.

Reply:
I'm not sure you can unless you do it in a separate component, since the component with the provider, Cytoscape, is used here directly. It didn't work when I tried it anyway, I can try again.

Reply:
Used the context in another component instead.